package.json

```diff
@@ -1,15 +1,58 @@
 {
   "name": "filmfinder",
   "version": "1.0.0",
-  "description": "v 0.0.1",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack --config=configs/webpack.production.config.js",
+    "start": "webpack-dev-server --open --config=configs/webpack.development.config.js",
+    "test": "jest",
+    "test-update": "jest --updateSnapshot",
+    "test-eslint": "eslint '**/*.{js,jsx}' --quiet"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/aleksandrbelik/filmFinder.git"
   },
-  "author": "",
-  "license": "ISC",
-  "homepage": "https://github.com/aleksandrbelik/filmFinder#readme"
+  "author": "Aleksandr Belik",
+  "contributors": [
+    "Danila Loginov"
+  ],
+  "license": "UNLICENSED",
+  "homepage": "https://github.com/aleksandrbelik/filmFinder#readme",
+  "devDependencies": {
+    "@babel/core": "^7.4.0",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "babel-jest": "^24.5.0",
+    "babel-loader": "^8.0.5",
+    "babel-plugin-module-resolver": "^3.2.0",
+    "babel-plugin-transform-export-extensions": "^6.22.0",
+    "css-loader": "^2.1.1",
+    "eslint": "^5.15.3",
+    "eslint-config-airbnb-base": "^13.1.0",
```

Comment:
Prefer `eslint-config-airbnb` instead of `eslint-config-airbnb-base`.

Reply:
@loginov-rocks fixed